apps/nameguard.io/package.json

```diff
@@ -12,7 +12,6 @@
     "@heroicons/react": "2.0.18",
     "@namehash/ens-utils": "workspace:*",
     "@namehash/ens-webfont": "workspace:*",
-    "@namehash/namekit-react": "workspace:*",
```

Comment:
We're not removing this dependency. I'm just removing a duplicate reference to this dependency. It's listed twice in this package.json.